packages/rocket-uploads3-store-event-aws-infrastructure/src/lambdas/index.ts

```diff
@@ -0,0 +1,44 @@
+import { AWSError } from 'aws-sdk'
+import { PutItemOutput } from 'aws-sdk/clients/dynamodb'
+
+export const handler = async (event: any): Promise<any> => {
+
+  const AWS = require('aws-sdk')
```

Comment:
Can't you import this as `import * as AWS from { aws-sdk }`? 😮

Reply:
@juanjoman I think we can yes, do we have any guidelines what is the preferred way for these imports? require vs from, * vs specific imports.
Asking because I don't really know them hehe :P

Reply:
There are some differences between how `import` and `require` load the libraries and also it depends on how the module was exported. https://blog.atomist.com/typescript-imports